app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_with(model: [current_user, @post, Comment.new], local: true) do |form| %>
+  <div class="field">
+    <%= form.label :text %>
+    <%= form.text_area :text %>
+  </div>
+  <%= form.submit 'Add Comment' %>
+<% end %>
```

Comment:
- [ ] Great job @mohammadnewton, there is something to have to fix before this PR is approved, According to the project requirements, there should be a form in the views/comments/new.html.erb file that accepts input for creating comments. Please address this. ✍🏻